pkg/security/apparmor/validate.go

```diff
@@ -43,25 +39,8 @@ type Validator interface {
 	ValidateHost() error
 }
 
-// NewValidator is in order to find AppArmor FS
-func NewValidator() Validator {
```

Comment:
I think we still need this function, it is called from the kubelet and we just have to remove the `getAppArmorFS()` call below.